src/organisms/MainNavBar.test.jsx

```diff
@@ -38,6 +46,12 @@ test('Left Dropdown menu click invokes SetState function to remove "show"', () =
   dropdownMenu.simulate('click', { preventDefault() {} });
   expect(mockSetState).toHaveBeenCalledWith({ leftMenu: '', rightMenu: '' });
 });
+test('Left Dropdown menu click invokes SetState function to remove "show"', () => {
+  const wrapper = setup('show', '');
+  const dropdownMenu = findByTestAttr(wrapper, 'leftdropdownMenu');
+  dropdownMenu.simulate('keydown', { keyCode: 13 });
```

Comment:
Key codes probably not worth testing

Reply:
I'll just leave the test there for now, doesn't hurt